openeo/Sentinel-3.ipynb

```diff
@@ -1,5 +1,93 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "96b5d47a-3d98-41a1-a3f1-ee1e2e511677",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/opt/conda/envs/openeo/lib/libtiff.so\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "140164313301568"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from rasterio import logging\n",
+    "\n",
+    "log = logging.getLogger()\n",
+    "log.setLevel(logging.ERROR)\n",
+    "\n",
+    "import warnings\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "\n",
+    "# for tiffwarning:\n",
+    "import ctypes\n",
+    "from ctypes.util import find_library\n",
+    "\n",
+    "# Find the path to the library we want to modify\n",
+    "thePath = find_library('tiff')     # try \"gdal\" instead of \"tiff\" too\n",
+    "print(thePath)\n",
+    "\n",
+    "# Get handle to it\n",
+    "theLib = ctypes.CDLL(thePath)\n",
+    "theLib.TIFFSetWarningHandler.argtypes = [ctypes.c_void_p]\n",
+    "theLib.TIFFSetWarningHandler.restype = ctypes.c_void_p\n",
+    "theLib.TIFFSetWarningHandler(None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "7aac38d4-6e6a-425d-aa9a-d170e8535798",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "                <script type=\"application/javascript\" id=\"jupyter_black\">\n",
+       "                (function() {\n",
+       "                    if (window.IPython === undefined) {\n",
+       "                        return\n",
+       "                    }\n",
+       "                    var msg = \"WARNING: it looks like you might have loaded \" +\n",
+       "                        \"jupyter_black in a non-lab notebook with \" +\n",
+       "                        \"`is_lab=True`. Please double check, and if \" +\n",
+       "                        \"loading with `%load_ext` please review the README!\"\n",
+       "                    console.log(msg)\n",
+       "                    alert(msg)\n",
+       "                })()\n",
+       "                </script>\n",
+       "                "
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import jupyter_black\n",
+    "%load_ext jupyter_black"
```

Comment:
I would not expose an extension like `jupyter_black` to the end user. It's unnecessary to impose this dependency on them

Reply:
This too wasn't supposed to be part of this cell.